malicious-token-contract-ml-py/src/findings.py

```diff
@@ -21,10 +23,15 @@ def __init__(
 
     def malicious_contract_creation(
         self,
-        anomaly_score: float,
+        chain_id: int,
         labels: list,
     ) -> Finding:
-        self.metadata["anomaly_score"] = anomaly_score
+        self.metadata["anomaly_score"] = calculate_alert_rate(
+            chain_id,
+            BOT_ID,
+            "SUSPICIOUS-TOKEN-CONTRACT-CREATION",
+            ScanCountType.TRANSFER_COUNT,
```

Comment:
Could you use `ScanCountType.CONTRACT_CREATION_COUNT` for the denominator?
https://github.com/forta-network/bot-alert-rate/blob/main/python/bot_alert_rate/calculate_alert_rate.py#L12